config/locales/en.yml

```diff
@@ -1,4 +1,17 @@
 ---
+0: Continue now and verify by mail, which takes <strong>5 to 10 days</strong>.
+1: Exit %{app_name} and try again later.
+10: October
+11: November
+12: December
+2: February
+3: March
+4: April
+5: May
+6: June
+7: July
+8: August
+9: September
```

Comment:
these need to be nested under something

Reply:
@zachmargolis I noticed I got something similar to this in another branch when running your `scripts/yml_fix_merge_conflicts` helper, so maybe something going on there?